GOVERNANCE.md

```diff
@@ -0,0 +1,46 @@
+# Process for becoming a maintainer
+
+## Your organization is not yet a maintainer
+
+* Express interest to the senior maintainers that your organization is interested in becoming a maintainer. Becoming a maintainer generally means that you are going to be spending substantial time (>25%) on Amundsen for the foreseeable future. Ultimately your goal is to become a senior maintainer that will represent your organization.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing senior maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 2-3 months of working together and making sure we see eye to eye, the existing senior maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid reasons are "blast radius", a large organization that is working on multiple unrelated projects, etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard" maintainer access. This will require 3 (senior) maintainers +1 vote and no -1 vote.
```

Comment:
```suggestion
* After 1-2 months, the existing senior maintainers will discuss granting "standard" maintainer access. The approval requires 3 (senior) maintainers +1 vote and no -1 vote.
```

Reply:
Should be `atleast` 3 senior maintainers approval right? Also, could you elaborate on how this would be carried out? Should the member apply to be a maintainer or is this more a decision by senior maintainer to initiate the process?

Reply:
Would the approval conversation happen over a email thread or would it be a poll? Do you think we should add some guidelines on how long should the poll be open (like min of 3 business or 1 week) as the poll cannot be closed as soon as we get 3 approvals right? We would have to provide sufficient time for all senior members to review ( and potentially do a -1 if required).

Reply:
could you elaborate on what is required for someone to vote -1? Also, could you elaborate on what is the process when someone is denied maintained status? Should they expect some communication around what they should next?